### PR TITLE
Initial seccomp docs

### PR DESCRIPTION
**Pull request by NissesSenap**
This feature haven't been merged yeat and shoulden't go live until we do.
Other docs should be merged soon and we should probably set this under the same header.

### CODE REVIEW

**Comment by NissesSenap** [comment]
Waiting for https://github.com/XenitAB/xenitab.github.io/pull/105 before this is merged.
Same goes for: https://github.com/XenitAB/gatekeeper-library/pull/68 and https://github.com/XenitAB/terraform-modules/pull/590

File: docs/xks/developer-guide/security.md [hunk 1]
```diff
@@ -9,3 +9,45 @@ Certain policies will always be enforced in the cluster as a guard rail to minim
 any request to create or update a Kubernetes resource will first have to pass through the policy check. If the resource in question does not comform to the policy it will be rejected
 by the API server when applying the change. Knowing this is important as certain features or options documented on the Internet may not be available or are restricted in the XKS service.
 This can include things like certain types of volume mounts, labeling requirements or container capabilities for a Pod.
+
+## Seccomp
+
+Secure computing mode or [seccomp](https://docs.docker.com/engine/security/seccomp/) for short.
+SeccompProfile is a pod wide securityContext setting and is a way to restrict which system calls a application can make inside a container.
+
+In XKF we have added a mutating webhook that automatically assigns the `RuntimeDefault` default profile to any container started.
```
Comment (phillebaba) [inline docs/xks/developer-guide/security.md:18]:
```suggestion
XKF is configured to mutate all Pods, which do not specify a seccomp profile, with the profiler `RuntimeDefault`.
```

**Review by phillebaba — COMMENTED** [review]
Added some recommended changes. Check if those are valid and move this section to the container security page and we should be all right.

**Comment by NissesSenap** [comment]
Good feedback @phillebaba , I have now moved the docs and we should be ready to merge.

File: docs/xks/developer-guide/container-security.md [hunk 2]
```diff
@@ -40,6 +40,9 @@ spec:
           drop:
             - NET_RAW
             - CAP_SYS
+  securityContext:
+    seccompProfile:
+      type: RuntimeDefault
```
[the remark this answers is not in the document]
Comment (phillebaba) [reply docs/xks/developer-guide/container-security.md:45]: I think it should be there. When I wrote container security i did not mean only container security contexts but rather topics related to runtime security compared to for example Kubernetes ingress security. The example at the top should contain all mutations that a user should expect to happen to their Pod.

**Review by phillebaba — APPROVED** [review]

**Pull request merged**